Helm lint command should be given the same variables as the upgrade or install one

## Issue

If `helm`'s linting command is not given all the variables used by the deployment, it will fail if the `_helpers.tpl` makes use of any of these variable. Which it may.

Example, with all the variables:
```
$ helm lint ./k8s/jobflow/ \
    --set=circleci.sha1=e1011e0757985241122f87ac6b61c4cee5cf0e0a \
    --set=image=jobteaser/jobdata:git-stats-e1011e0757985241122f87ac6b61c4cee5cf0e0a 
==> Linting ./k8s/jobflow/
[INFO] Chart.yaml: icon is recommended

1 chart(s) linted, no failures
```
With some missing variables:
```
$ helm lint ./k8s/jobflow/ \
    --set=image=jobteaser/jobdata:git-stats-e1011e0757985241122f87ac6b61c4cee5cf0e0a 
==> Linting ./k8s/jobflow/
[INFO] Chart.yaml: icon is recommended
[ERROR] templates/: render error in "jobflow/templates/deployment.yaml": template: jobflow/templates/deployment.yaml:24:70: executing "jobflow/templates/deployment.yaml" at <.Values.circleci.sha1>: nil pointer evaluating interface {}.sha1

Error: 1 chart(s) linted, 1 chart(s) failed
```

## Work-around

For now one can disable linting.

## Fix

Would gladly offer a PR, though I am not sure yet how one could "macro" this to not have to repeat those arguments in several places in the orb. The stupid fix is easy enough though, if the smarter one is not possible.

--- a/orbs/helm/orb.yml
+++ b/orbs/helm/orb.yml
@@ -66,6 +66,7 @@ commands:
             helm lint \
               -f <<parameters.helm_value_file_path>> \
               --set=circleci.sha1=$CIRCLE_SHA1 \
+              --set=circleci.branch=$CIRCLE_BRANCH \
               <<parameters.helm_chart_path>>
 
   expand_helm_templates:
@@ -96,6 +97,7 @@ commands:
               -n <<parameters.helm_release_name>> \
               -f <<parameters.helm_value_file_path>> \
               --set=circleci.sha1=$CIRCLE_SHA1 \
+              --set=circleci.branch=$CIRCLE_BRANCH \
               --output-dir /tmp/helm-templates/<<parameters.environment>> \
               <<parameters.helm_chart_path>>
       - store_artifacts:

